src/components/GuidedInstallTile.js

```diff
@@ -74,8 +74,9 @@ const GuidedInstallTile = () => {
               }
             `}
           >
-            Install the New Relic agent with a single command line and start
-            monitoring your log and infrastructure data in real time.
+            Many engineers start here. You'll install an agent with a single
+            command line and start monitoring your log and infrastructure data
+            in real time.
```

Comment:
Can we check with the writing team on this? "a single command line" doesn't make a lot of sense. Maybe we can change it to something like:
```suggestion
            Many engineers start here. You'll install an agent with a single
            command and start monitoring your log and infrastructure data
            in real time.
```

Reply:
all the final copy is in the FIGMA file but I agree to this change. I'll note it in the FIGMA file we changed this.